lib/private/Files/Filesystem.php

```diff
@@ -492,9 +494,12 @@ public static function isBlacklisted($data) {
 	public static function isFileBlacklisted($filename) {
 		$filename = self::normalizePath($filename);
 
-		$blacklist = \OC::$server->getConfig()->getSystemValue('blacklisted_files', ['.htaccess']);
+		if (!self::$blacklist) {
```

Comment:
```suggestion
		if (self::$blacklist === null) {
```
Blacklist may be an empty array, no?